src/pages/appflow/destinations/apple.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Apple App Store'
+previousText: 'Deploy To Appstore'
+previousUrl: '/docs/appflow/destinations/intro'
+nextText: 'Deploy to Google Play store'
+nextUrl: '/docs/appflow/destinations/google'
+---
+
+The Apple App Store destination allows you to upload your finished binaries to the App Store Connect service for iOS devices.
+
+
+![/docs/assets/img/appflow/apple-add-destination.png](/docs/assets/img/appflow/apple-add-destination.png)
+
+For this destination you will need the following:
+
+(Not part of the destination, but to build a binary that you can upload to the Apple App Store you will need a production Security Profile configured in App Flow, details [here](https://ionicframework.com/docs/appflow/package/credentials#ios-credentials))
```

Comment:
I would suggest putting this in a warning box as a prerequisite to call attention to it. See example here: https://github.com/ionic-team/ionic-docs/blame/master/src/pages/appflow/deploy/setup.md#L16